### ISSUE TITLE
version 3.5.4 and 3.5.5 throws "System.InvalidOperationException" when creating new SLDocument()

### ISSUE THREAD

**Issue by selganor74** [thirstyape/spreadsheetlight-cross-platform#3]
I ran into the problem after upgrading from 3.5.1 to 3.5.5. Reverting back to 3.5.1 solved the problem, so I tried to go up one patch at a time, and found that 3.5.3 still worked correctly, but 3.5.4 and 3.5.5 don't . 

This is the offending code:

```
using var sl = new SLDocument(pathToXlsx, checklistWorksheetName);
```

And this is the stack trace:

```
System.InvalidOperationException: Error in implicit conversion. Cannot convert null object.
   at DocumentFormat.OpenXml.EnumValue`1.op_Implicit(EnumValue`1 value)
   at SpreadsheetLight.SLCell.FromCell(Cell c)
   at SpreadsheetLight.SLDocument.LoadSelectedWorksheet()
   at SpreadsheetLight.SLDocument.OpenExistingSpreadsheet(String SheetNameOnOpen)
   at SpreadsheetLight.SLDocument..ctor(String SpreadsheetFileName, String SheetNameOnOpen)

```

**Comment by thirstyape** [comment]
Ok, I believe I have found it. 

I typically use `<Nullable>enable</Nullable>` in my projects to prevent this kind of thing; however as this is a big project that I have forked from the original enabling that would get a little nutty. 

Working to resolve now.

**Comment by thirstyape** [comment]
Fixed in 3.5.6
